dev/RadioButtons/RadioButtons.cpp

```diff
@@ -61,6 +62,20 @@ void RadioButtons::OnListViewSelectionChanged(const winrt::IInspectable& sender,
     m_selectionChangedEventSource(sender, args);
 }
 
+void RadioButtons::OnListViewGettingFocus(const winrt::IInspectable& /*sender*/, const winrt::GettingFocusEventArgs& args)
+{
+    if (auto listView = m_listView.get())
```

Comment:
>auto [](start = 8, length = 4)

this should be auto&&

Reply:
This matches all the other uses in the file.
I can go update them all though.

Reply:
A quick search shows only TeachingTip is using this style. Is this a recent change in recommended style?

My main use of auto&& in the past was in Cx due to... Cx being Cx. If it's correct here too I'm happy to comply.

Reply:
Yes, this is updated guidance since we started proliferating this pattern. In short, use auto&& when the local represents an alias of an intermediate calculation that is used either to avoid calling a function multiple times or to improve readability. In particular, tracker_ref.get() returns a const& and auto would make a copy of it. auto&& produces a local of the "right" type for this scenario.